src/org/labkey/targetedms/parser/SkylineDocumentParser.java

```diff
@@ -2054,6 +2105,12 @@ else if(XmlUtil.isStartElement(reader, evtType, TRANSITION_PEAK))
             else if (XmlUtil.isStartElement(reader, evtType, RESULTS_DATA)) {
                 chromInfoList.addAll(readTransitionResultsData(reader));
             }
+            else if (XmlUtil.isStartElement(reader, evtType, TRANSITION_LIB_INFO))
+            {
+                transition.setRank(Integer.parseInt(reader.getAttributeValue(null, "rank")));
```

Comment:
This will blow up if the attribute is null. I'm guessing we want to be null-tolerant

Reply:
Should we use XmlUtil.readIntegerAttribute() instead?